src/org/labkey/test/util/DeferredErrorCollector.java

```diff
@@ -365,9 +418,45 @@ public int getErrorCount()
         }
 
         @Override
-        public String takeScreenShot(String snapShotName)
+        public String takeScreenShot(String screenshotName)
```

Comment:
Same comment above about JavaDoc comments and public interfaces.

Reply:
The overridden method is documented. This is in the private wrapper class used for the `fatal` and `withScreenshot` error collectors.